codeforamerica/vita-min
# GYR1-489 Dashboard Page Stub and Feature Flag

## [GYR1-489](https://codeforamerica.atlassian.net/browse/GYR1-489)
## Is PM acceptance required?
## What was done?

### Created new Feature flag `hub_dashboard` 
![image](https://github.com/codeforamerica/vita-min/assets/17094895/4c59f18b-a61a-415a-b3c3-183c73543b17)

### Created Left menu item for dashboard 
![image](https://github.com/codeforamerica/vita-min/assets/17094895/bd069ecd-7335-4bf4-ae55-dc3e82fa5e05)

### Create dashboard stub controller
![image](https://github.com/codeforamerica/vita-min/assets/17094895/ac2c2338-0fa1-4713-9d4f-a143160cd9b0)

## How to test?
- Make sure the new dashboard menu item does not appear when the feature flag is disabled
- Make sure that the new dashboard menu item appears when the feature flag is enabled
- For a Coalition Lead user (lemon@example.com)...
  - Make sure that the dashboard filter includes only sites to which the user has access (`Oregano Org`, `Orangutan Organization`, `Liberry Site`)
- For an Organization Lead user (skywalker@example.com)...
  - Make sure that the dashboard filter includes only sites to which the user has access (`Oregano Org`, `Liberry Site`)
- For a Site Coordinator User (cucumber@example.com)...
  - Make sure that the dashboard filter includes only sites to which the user has access (`Liberry Site`)
- For a Greeter User (greeter@example.com)...
  - The dashboard menu item should not be displayed.

[GYR1-489]: https://codeforamerica.atlassian.net/browse/GYR1-489?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
